test/libsolidity/util/TestFileParser.cpp

```diff
@@ -107,6 +108,13 @@ vector<solidity::frontend::test::FunctionCall> TestFileParser::parseFunctionCall
 						if (lowLevelCall)
 							call.kind = FunctionCall::Kind::LowLevel;
 
+						// Treat all calls to functions containing '.' as calls to builtin functions.
```

Comment:
Is this special case really needed?

Reply:
I think yes, because normally the full function signature is needed to be present. But this is not the case for builtins.

Reply:
How is this connected to function signatures? Can't you just match the name with the list of builtins to see if it is a builtin?

Reply:
Ah true, I thought you meant somehow that this special case of parsing is not needed. Good idea!